pkg/snapshot/filesystem.go

```diff
@@ -1101,3 +1114,28 @@ func IsFileSystemMinioDisabled(kotsadmNamespace string) (bool, error) {
 
 	return false, nil
 }
+
+func IsMinIOImagePresent() (bool, error) {
```

Comment:
this should take a clientset instead of making one itself for ease of testing

Reply:
see https://github.com/replicatedhq/kots/blob/a31b5e55b657b7c1c7c7824be2010ccb96a7e923/pkg/auth/auth.go#L30